### ISSUE TITLE
Library does not preserve HTML tags in strings when rendering

### ISSUE THREAD

**Issue by kmayoral** [B3nedikt/restring#85]
Thanks for a great library and for taking up the challenge of maintaining and fixing issues with the original implementation as well as the ViewPump library!

I was testing out the library today and noticed that strings with HTML content do not appear to be preserved when being rendered through the library, even if the string itself is not replaced with a newer value. Not only is the formatting lost, but if the string in question features an ```<a>``` tag, the ability to click on that link is no longer possible.

To reproduce, I added the following in the restring.example project's ```strings.xml```:
```
<string name="example_html" translatable="false"><a href="https://www.github.com">Github URL link</a> example</string>
```

And to ```fragment_main.xml```:
```
    <TextView
      android:id="@+id/htmlTextView"
      android:layout_width="wrap_content"
      android:layout_height="wrap_content"
      android:text="@string/example_html"
      android:textSize="16sp"
      app:layout_constraintTop_toBottomOf="@+id/toolbar"
      app:layout_constraintLeft_toLeftOf="parent"
      app:layout_constraintRight_toRightOf="parent" />
```

Expected:
![Screen Shot 2021-03-08 at 5 13 59 PM](https://user-images.githubusercontent.com/801410/110403320-c31ad300-8031-11eb-8cbd-6ac45ae68443.png)

Actual:
![device-2021-03-08-171528](https://user-images.githubusercontent.com/801410/110403415-e8a7dc80-8031-11eb-8d02-fa749d712a0a.png)

Let me know if you need any additional details and thank you again for the library!

**Comment by B3nedikt** [comment]
You're welcome, I am glad you like it :)

I just fixed this issue with Reword 4.0.1, just update and it should work :)

```groovy
// Replace bundled strings dynamically
implementation 'dev.b3nedikt.restring:restring:5.1.2'

// Intercept view inflation
implementation 'dev.b3nedikt.viewpump:viewpump:4.0.7'

// Allows to update the text of views at runtime without recreating the activity
implementation 'dev.b3nedikt.reword:reword:4.0.1'
```

**Comment by kmayoral** [comment]
Wow that was fast, thank you! Tested and verified on my side, appreciate the quick help on this!!

Note to future readers: if you plan on supplying replacement strings containing styled HTML, you should make sure to explicitly convert your string to a SpannedString like so: 
```
HtmlCompat.fromHtml("<a href="https://www.github.com">Github URL link</a> example", HtmlCompat.FROM_HTML_MODE_LEGACY)
```
before you pass it into the `Restring.putStrings()` method. This will ensure that the spans are styled correctly when restring/rewording.
